.github/workflows/release.yml

```diff
@@ -18,17 +18,17 @@ jobs:
       - name: Use Node.js 16.x
         uses: actions/setup-node@v1
         with:
-          version: 16.x
```

Comment:
[nit] We could add `.nvmrc` and use it both for dev env and our CI as well.

Reply:
addressed

Reply:
Thanks! I don't see the `.nvmrc` committed though 🤔

Reply:
Ignore my comment, it already exists 🙏